src/PrestaShopBundle/Resources/config/services/adapter/mail_template.yml

```diff
@@ -9,9 +9,6 @@ services:
       - '@prestashop.core.mail_template.variables_builder'
       - '@prestashop.core.hook.dispatcher'
     calls:
-      - method: 'addTransformation'
```

Comment:
Here I disable completely the `css_inline` transformation.
This is not correct, because the `classic` email theme requires this transformation.

Can someone suggest me how to conditionally call `addTransformation` here.